go/.env.example

```diff
@@ -0,0 +1,10 @@
+DB_USER=root
+DB_PASS=
+DB_NAME=go_db
+DB_HOST=localhost
+DB_PORT=3306
+OMISE_PUB_KEY=pkey_test_sssss
```

Comment:
This doesn't match the README. Nor does it match the other examples.

Reply:
sure, I will check

Reply:
> This doesn't match the README.

I can not find that does not match the README.

it looks same,
![Uploading Screenshot 2023-11-14 at 14.18.31.png…]()

Reply:
> This doesn't match the README.

I can not find that does not match the README.

it looks same.

<img width="1507" alt="Screenshot 2023-11-14 at 14 18 31" src="https://github.com/omise/examples/assets/100895497/cf35a9ba-b803-49c7-a547-d08c2a579163">